code/src/components/Task.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useDispatch } from 'react-redux'
+
+import { tasks } from 'reducers/tasks'
+import { CheckBox } from './CheckBox'
+import trashCan from '../assets/trashcan.png'
+
+
+export const Task = ({ item }) => {
+  const dispatch = useDispatch()
+
+  const handleDelete = (id) => {
+    dispatch(tasks.actions.removeTask(id))
+  }
+
+  const handleCheckboxClick = () => {
+    dispatch(tasks.actions.setChecked(item.id))
+  };
+
+
+  return (
+    <TaskContainer>
+      <CheckBox
+        isChecked={item.complete}
+        onChangeHandler={handleCheckboxClick}
+      />
+      <TaskText line={item.complete ? 'line-through' : null}>{item.text}</TaskText>
```

Comment:
Hmm, I might be tired, but I think that the CSS should be in the styling below, and this part rather be a variable? Just to make the code a bit easier to understand 😊